src/tests/test_ct_utils.cpp

```diff
@@ -0,0 +1,180 @@
+/*
+* (C) 2018,2024 Jack Lloyd
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include "tests.h"
+#include <botan/internal/ct_utils.h>
+#include <botan/internal/fmt.h>
+#include <functional>
+
+namespace Botan_Tests {
+
+class CT_Mask_Tests final : public Test {
```

Comment:
I moved the CT tests to a new file as `test_utils.cpp` is getting quite large. The Mask and Choice tests are unchanged